packages/lib-classifier/src/plugins/drawingTools/components/Mark/Mark.spec.js

```diff
@@ -233,73 +318,69 @@ describe('Drawing tools > Mark', function () {
       )
     }
 
-    describe('with finished mark and no subTaskVisibility', function () {
+    describe('when subtasks are closed', function () {
       let newMark
 
       before(function () {
         window.scrollTo.resetHistory()
         newMark = pointTool.createMark()
```

Comment:
This mark has no tasks, so why does the tasks popup open when it finishes? 🤔

Reply:
Never mind. The default point tool for these tests, defined at the top of the file, has a text task.